frontend/src/queries/schema/schema-general.ts

```diff
@@ -2395,12 +2395,12 @@ export type FileSystemIconType =
     | 'cohort'
     | 'group'
     | 'insight_funnel'
-    | 'insight_trend'
+    | 'insight_trends'
     | 'insight_retention'
-    | 'insight_user_path'
+    | 'insight_paths'
     | 'insight_lifecycle'
     | 'insight_stickiness'
-    | 'insight_hogql'
+    | 'insight_hog'
```

Comment:
Can we have a `/` instead of a `_` to split the type? This would work better with the existing file system models --> we already have types like `hog_function/destination`, etc, in the database.